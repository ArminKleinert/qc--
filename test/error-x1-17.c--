
register bits8 c;

section "data" 
{
    x:
    bits32[4]; 
}

foo () {
        
        jump 1 targets c; 
}



