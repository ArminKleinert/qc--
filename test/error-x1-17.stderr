file error-x1-17.c--, line 12, characters 9-26 Error: c is not a label
